README.md

```diff
@@ -49,8 +49,9 @@ $ stack build   # Compile Courseography and all Haskell dependencies (this will
 
 #### Parsing and Generation
 
-1. Create database file for an parse prerequisite graph `$ stack run graphs`
-2. Parse course information `$ stack run database`
+1. Create the database file `$ stack exec courseography database-setup`
```

Comment:
You can use `stack run database-setup` here to be consistent with the other commands (both `stack run` and `stack exec courseography` do the same thing for our purposes)